index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Paris</title>
+
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles/reset.css">
+  <link rel="stylesheet" href="./styles/weather_widget.css">
+</head>
+<body>
+<aside id="weather-widget">
+  <div id="main-part" class="clearfix">
+    <div id="current-weather">
+      <p id="current-temp">12 °</p>
+      <p id="current-place">Paris, France</p>
+      <div id="current-wind">
+        <div id="wind-icon"></div>
```

Comment:
BEM doesn't use tag and `id` selectors. The styles of blocks and elements are described through `class` selectors.
It's better to replace `id` with `class`.
FYI: https://ru.bem.info/methodology/css/

Reply:
[Fixed](https://github.com/js-training-sep-2020/hw-1/pull/5/commits/7c5e3b40dd326e1979b71ef9b5b65a4b8422b18a)